File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/order/exception/NotFreeOrderException.java
package band.gosrock.domain.domains.order.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class NotFreeOrderException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new NotFreeOrderException();

    private NotFreeOrderException() {
        super(OrderErrorCode.ORDER_NOT_FREE);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/order/exception/NotFreeOrderException.kt
package band.gosrock.domain.domains.order.exception

import band.gosrock.common.exception.DuDoongCodeException

class NotFreeOrderException private constructor() : DuDoongCodeException(OrderErrorCode.ORDER_NOT_FREE) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = NotFreeOrderException()
    }
}
